src/Aspire.Hosting/Extensions/ResourceBuilderExtensions.cs

```diff
@@ -328,6 +328,38 @@ private static void ApplyEndpoints<T>(this IResourceBuilder<T> builder, IResourc
         return builder.WithAnnotation(annotation);
     }
 
+    /// <summary>
+    /// Exposes an HTTP endpoint on a resource. This binding reference can be retrieved using <see cref="GetEndpoint{T}(IResourceBuilder{T}, string)"/>.
+    /// The binding name will be "http" if not specified.
+    /// </summary>
+    /// <typeparam name="T">The resource type.</typeparam>
+    /// <param name="builder">The resource builder.</param>
+    /// <param name="hostPort">The host port.</param>
```

Comment:
I think it would be helpful to add details to the nullable parameter doc-comments about what the behavior is if the parameter is omitted/null.